modules/claims_api/config/routes.rb

```diff
@@ -33,6 +33,8 @@
   end
 
   namespace :docs do
+    match '/metadata', to: 'metadata#index', via: [:get]
```

Comment:
Could we move this up a namespace? Metadata to me contains more than just docs, (thinking in the future having healthchecks and whatnot)

Reply:
Done